### PR TITLE
Shipping Labels: enable remaining M4 features

### PR DESCRIPTION
**Pull request by hichamboushaba**
This PR enables remaining shipping labels M4 features, and removes the feature flag:
- Enables package creation.
- Enables adding payment methods.
- Enable multiple packages.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreationFeatures.kt [hunk 1]
```diff
@@ -1,9 +1,7 @@
 package com.woocommerce.android.ui.orders.shippinglabels.creation
 
-import com.woocommerce.android.util.FeatureFlag
-
 object ShippingLabelCreationFeatures {
-    val CAN_CREATE_PAYMENT_METHOD = FeatureFlag.SHIPPING_LABELS_M4.isEnabled()
-    val CAN_CREATE_PACKAGE = FeatureFlag.SHIPPING_LABELS_M4.isEnabled()
+    const val CAN_CREATE_PAYMENT_METHOD = true
```
Comment (nbradbury) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreationFeatures.kt]: Are these three still necessary?

**Comment by hichamboushaba** [comment]
@designsimply something to note when writing the test notes for this PR, adding payment methods through the app will target the production server, so to test the feature, you need to disable the connect-server staging plugin (this step in the testing guide p91TBi-3xD-p2#staging) in your store, but avoid to purchase labels after doing so, as it will charge your live credit card.

Please let me know if something is not clear in the process 🙂

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreationFeatures.kt (hunk 1 above)
Comment (hichamboushaba) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreationFeatures.kt]: I'm wondering the same thing 🙂, I think you are right, maybe it's cleaner if I just removed the whole file, and passed `true` directly in the repository.

Comment (hichamboushaba) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/shippinglabels/creation/ShippingLabelCreationFeatures.kt]: Done in 7965183

**Review by nbradbury — APPROVED** [review]
Looks good! :shipit:

**Pull request merged**